policy.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This is probably not meant to be present

Reply:
ops you are correct, removed!